test/epgsql_cast.erl

```diff
@@ -40,9 +40,9 @@ await_connect(Ref) ->
         {C, Ref, connected} ->
             {ok, C};
         {_C, Ref, Error = {error, _}} ->
-            Error;
-        {'EXIT', _C, _Reason} ->
-            {error, closed}
+            Error
+    after 5000 ->
```

Comment:
Is there any reason to make this configurable?

Reply:
@davidw it's possible - there is a `timeout` connect option, we may use it for this. But it's just a test module, not a public API and this timeout means failed test usually. Do you think somebody may need to change that?

Reply:
@davidw ok, I'll just do that using mentioned `timeout` connect option

Reply:
Done